pkg/kubelet/kubelet_test.go

```diff
@@ -1622,6 +1625,8 @@ func TestDoesNotDeletePodDirsForTerminatedPods(t *testing.T) {
 
 func TestDoesNotDeletePodDirsIfContainerIsRunning(t *testing.T) {
 	testKubelet := newTestKubelet(t, false /* controllerAttachDetachEnabled */)
+	go testKubelet.kubelet.podKiller.PerformPodKillingWork()
+	defer testKubelet.kubelet.podKiller.Close()
```

Comment:
Was this change just missed the last time we updated this test?

Reply:
Some tests do not rely on the podKiller to be running. There needs to be a separate cleanup pr to put ```go testKubelet.kubelet.podKiller.PerformPodKillingWork()``` into newTestKubelet().

Reply:
Actually, I'll add it to this one. It's relatively small.